recipes/txl-cell/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "txl-cell" %}
+{% set version = "0.2.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/txl_cell-{{ version }}.tar.gz
+  sha256: 66a2359a7a9dec70c088e0d00e5601be3116e40c81ccc6de0ed964efc6eda3a9
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  host:
+    - python >=3.10
+    - pip
+    - hatchling
+  run:
+    - python >=3.10
+    - txl
+    - pycrdt >=0.8.11,<0.9.0
+
+test:
+  # `pip check` was disabled, see: https://github.com/conda-forge/staged-recipes/pull/26869
```

Comment:
Should probably open an issue upstream and xref it here. The conversation on stage-recipes helps understand the problem but doesn't track it.